offline/packages/bcolumicount/StreamingBcoLumiReco.cc

```
@@ -0,0 +1,310 @@
+    {
+      m_rawgl1scaler = packet->lValue(0, 0);
+    }
+
+    delete packet;
+
+    if (Verbosity() > 2)
+    {
+      if (!syncobject)
+      {
+        std::cout << PHWHERE << " SyncObject missing" << std::endl;
+        delete packet;
```

Cppcheck (2.20.0)

[error] Memory pointed to by 'packet' is freed twice.

(doubleFree)